mdx_oembed/endpoints.py

```diff
@@ -6,6 +6,14 @@
     'https?://youtu.be/*',
 ])
 
+SLIDESHARE = oembed.OEmbedEndpoint('http://www.slideshare.net/api/oembed/2', [
+    'http://www.slideshare.net/*/*',
```

Comment:
I'm fine with this. Thanks for pointing that repo out. I like the idea of pulling in the current list of providers from this source but I'm wary of making another dependency and/or making network requests every time this gets used.